src/Sulu/Bundle/MediaBundle/Controller/CollectionController.php

```diff
@@ -154,11 +158,12 @@ public function getAction($id, Request $request)
 
         try {
             $locale = $this->getRequestParameter($request, 'locale', true);
-            $depth = \intval($request->get('depth', 0));
+            $depth = \intval($this->getRequestParameter($request, 'depth', false, 0));
```

Comment:
getRequestParameter does the same as `$request->get` just with a fake string return type.